.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "go.lintTool": "golangci-lint",
   "go.useLanguageServer": true,
   "go.languageServerExperimentalFeatures": {
-    "diagnostics": false,
+    "diagnostics": false
```

Comment:
these changes are linter related, we might want to remove them :-)

Reply:
I do see that this "fix" is not really relevant, but I doubt that I am the only one who will open the settings.json and do formatting subconsciously. This will pop up later. I think leaving it fixed is a better option.